starter-code/src/app/app.module.ts

```diff
@@ -3,18 +3,32 @@ import { NgModule } from '@angular/core';
 import { FormsModule } from '@angular/forms';
 import { HttpModule } from '@angular/http';
 
+import { Routes, RouterModule } from '@angular/router';
+
 import { AppComponent } from './app.component';
+import { MoviesService } from './services/movies.service';
+import { MyHomeComponent } from './components/my-home/my-home.component';
+import { MyMovieComponent } from './components/my-movie/my-movie.component';
```

Comment:
The exercise says to name the page as `MyHomeComponent` and `MyMovieComponent` but since are pages, is a good practice name it as `HomePageComponent` and `MoviePageComponent`